metaserver/src/main/scala/scala/meta/languageserver/providers/CodeActionProvider.scala

```diff
@@ -1,13 +1,24 @@
 package scala.meta.languageserver.providers
 
+import scala.meta.languageserver.WorkspaceCommand.ScalafixUnusedImports
 import com.typesafe.scalalogging.LazyLogging
 import langserver.messages.CodeActionRequest
 import langserver.messages.CodeActionResult
+import langserver.types.Command
+import langserver.types.Diagnostic
 import play.api.libs.json.Json
 
 object CodeActionProvider extends LazyLogging {
   def codeActions(request: CodeActionRequest): CodeActionResult = {
-    logger.info(Json.prettyPrint(Json.toJson(request)))
-    CodeActionResult(Nil)
+    val removeUnusedImports = request.params.context.diagnostics.collectFirst {
+      case Diagnostic(_, _, _, Some("scalac"), "Unused import") =>
+        // This command doesn't seem to do much
```

Comment:
This comment confuses me.

Reply:
It's a leftover comment before I made it remove unused imports, comment removed